src/cobalt_strike.yml

```diff
@@ -0,0 +1,60 @@
+---
+- hosts: all
+  name: Install Cobalt Strike
+  become: yes
+  become_method: sudo
+  roles:
+    - openjdk
+    - role: cobalt_strike
+      vars:
+        bucket_name: "{{ build_bucket }}"
+    - server_setup
+  tasks:
+    - name: Install thunderbird mail client
```

Comment:
As long as we're fixing case issues here: `thunderbird` -> `Thunderbird`

Reply:
See commit f6ae79c.